packages/charts/src/charts/pie/pie-slice.component.tsx

```diff
@@ -66,6 +67,28 @@ const PieSlice: FC<Props> = ({
   const [x, y] = activePosition;
   const [labelX, labelY] = labelPosition;
 
+  const [isDelayed, setDelay] = useState(false);
+  const [isActive, setActive] = useState(false);
+
+  const activeControls = useAnimation();
+  const activeVariants = {
```

Comment:
For better readability of `render` method could You please move `activeVariants` out of method and change it fo function:
```
const activeVariants = (x: number, y: number) => Variants
```